### ISSUE TITLE
jira: datetime comparisons cause changelog on each pull

### ISSUE THREAD

**Issue by TheKevJames** [ralphbean/bugwarrior#599]
Every time I run `bugwarrior-pull` while configured against Jira, each of my issues prints a line similar to the following:

```
INFO:bugwarrior.db:Updating task 55a3dc8c-f557-4d0d-82a7-5a231c1b5a32, Water the dog; entry: datetime.datetime(2018, 4, 20, 0, 55, 42, tzinfo=tzutc()) -> datetime.datetime(2018, 4, 19, 17, 55, 42, 904000, tzinfo=tzoffset(None, -25200))
```

The later `datetime` is the result in my local timezone (UTC-7).

My impression is that this is recognized as a change since [the JiraIssue.get_entry method converts the date to UTC and removes the microseconds](https://github.com/ralphbean/bugwarrior/blob/develop/bugwarrior/services/jira.py#L128).

**Comment by ryneeverett** [comment]
I think this is a duplicate of #450 which we've already fixed in the `develop` branch but hasn't seen a release yet. If you try the `develop` branch and still experience this issue I'd be happy to reopen.

**Comment by TheKevJames** [comment]
Confirmed fixed in [release 1.6.0](https://github.com/ralphbean/bugwarrior/releases/tag/1.6.0). Thanks @ryneeverett !
